documentation/book/oauth/proc-oauth-server-config.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following module:
+//
+// con-oauth-config.adoc
+
+[id='proc-oauth-server-config-{context}']
+= Configuring an {oauth} authorization server
+
+This procedure describes how to configure an authorization server for integration with {ProductName}.
+
+Depending on the chosen xref:con-oauth-authentication-client-options-{context}[authentication flow], you can use the authorization server to:
+
+* Configure authentication for Kafka brokers
+* Configure and authorize clients
+* Configure users and roles
+* Obtain access and refresh tokens
+
```

Comment:
I would remove this sentence, and replace it with:
>>>>>
Preparing authorization server to work with Strimzi involves the following steps:
* Configuring 'kafka-broker' client
* Configuring clients for each Kafka client component of your application

Note: 'Kafka client' or 'Kafka client component' is an application, that conects to Kafka brokers. A 'client' on authorization server is a configuration that includes client id, client secret, and various parameters controlling interaction of application components with authorization server. Kafka brokers, and all Kafka clients need their own  'client' configuration at authorization server.
<<<<<

For now we don't yet support authorization so we don't need to mention users, and roles here. Optionally, obtaining long-lived access and refresh tokens ahead of time is not really a part of 'configuring the server' but more a part of  'deploying your Kafka client components'.